hbase_sql/src/main/java/com/splicemachine/hbase/BackupUtils.java

```diff
@@ -340,4 +343,11 @@ private static boolean backupTimedout() throws Exception {
         }
         return timedout;
     }
+
+    public static boolean incrementalBackupEnabled() {
+
+        Configuration conf = HConfiguration.unwrapDelegate();
+        String fileCleaners = conf.get("hbase.master.hfilecleaner.plugins");
+        return fileCleaners.contains("SpliceHFileCleaner");
```

Comment:
Incremental backup is enabled via the following:
hbase.master.hfilecleaner.plugins = com.splicemachine.hbase.SpliceHFileCleaner,
org.apache.hadoop.hbase.master.cleaner.TimeToLiveHFileClean

Should we check whether both are set instead of just "SpliceHFileCleaner"?

Reply:
TimeToLiveHFileCleaner is the default HBase file cleaner, which is not related to incremental backup

Reply:
Got it. Thanks.